src/NextCloud/players/NextCloudEmbed.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/media-has-caption */
```

Comment:
See https://taskman.eionet.europa.eu/issues/152940#note-29

Reply:
to be added later